include/wx/math.h

```diff
@@ -164,9 +164,21 @@ inline int wxRound(float x)
 
 inline int wxRound(long double x) { return wxRound(double(x)); }
 
+// For compatibility purposes, define wxRound() overloads for integer types
+// too, as this used to compile with wx 3.0.
+#if WXWIN_COMPATIBILITY_3_0
+
 wxDEPRECATED_MSG("rounding an integer is useless")
 inline int wxRound(int x) { return x; }
 
+wxDEPRECATED_MSG("rounding an integer is useless")
+inline int wxRound(short x) { return x; }
```

Comment:
`short` seems about as likely as `char`, I really doubt we need this one.

Reply:
OTOH it doesn't cost us anything, so I'd rather just add it than worry about/discuss whether we need it...